package.json

```diff
@@ -30,6 +30,7 @@
     "gulp-cache": "^0.2.10",
     "gulp-concat": "^2.6.0",
     "gulp-filter": "^3.0.1",
+    "gulp-if": "^2.0.0",
```

Comment:
This is a really nice solution, I can see gulp-if coming in pretty handy in future